apps/rush/src/utilities/test/PublishUtilities.test.ts

```diff
@@ -150,9 +150,11 @@ describe('sortChangeRequests', () => {
 
 describe('isRangeDependency', () => {
   it('can test ranges', () => {
+    /* tslint:disable:no-unused-expression */
     expect(PublishUtilities.isRangeDependency('>=1.0.0 <2.0.0')).is.true;
     expect(PublishUtilities.isRangeDependency('1.0.0')).is.false;
     expect(PublishUtilities.isRangeDependency('^1.0.0')).is.false;
     expect(PublishUtilities.isRangeDependency('~1.0.0')).is.false;
+    /* tslint:enable:no-unused-expression */
```

Comment:
Yeah it's sorta annoying.

---
In reply to: [129390566](https://github.com/Microsoft/web-build-tools/pull/276#discussion_r129390566) [](ancestors = 129390566)